src/components/VolumeViewer.jsx

```diff
@@ -12,21 +13,19 @@ import {
 import Controls from "./controls/Controls.jsx";
 import AframeScene from "./AframeScene.jsx";
 
-function VolumeViewer(props) {
-  const {
-    className,
-    style,
-    colorMap,
-    colorMaps,
-    controlsVisible,
-    model,
-    transferFunction,
-    useDefaultColorMaps,
-    useTransferFunction,
-  } = props;
-
+function VolumeViewer({
+  className,
+  style,
+  colorMap,
+  colorMaps,
+  controlsVisible,
+  model,
+  transferFunction,
+  useDefaultColorMaps,
+  useTransferFunction,
+}) {
   const [state, setState] = useState({
-    colorMap: colorMap,
+    colorMap: colorMap ? colorMap : DEFAULT_COLOR_MAP,
```

Comment:
could use the nullish coallescer here
```suggestion
    colorMap: colorMap ?? DEFAULT_COLOR_MAP,
```

Reply:
Ouu I didn't know that was a thing, neat JS really is full of neat little tricks